core/src/mindustry/logic/LogicOp.java

```diff
@@ -4,24 +4,32 @@
 import arc.util.*;
 
 public enum LogicOp{
+    //section arithmetic
     add("+", (a, b) -> a + b),
     sub("-", (a, b) -> a - b),
     mul("*", (a, b) -> a * b),
     div("/", (a, b) -> a / b),
     idiv("//", (a, b) -> Math.floor(a / b)),
     mod("%", (a, b) -> a % b),
+    pow("^", Math::pow),
+    //section boolean logic
     equal("==", (a, b) -> Math.abs(a - b) < 0.000001 ? 1 : 0, (a, b) -> Structs.eq(a, b) ? 1 : 0),
-    notEqual("not", (a, b) -> Math.abs(a - b) < 0.000001 ? 0 : 1, (a, b) -> !Structs.eq(a, b) ? 1 : 0),
+    notEqual("=/=", (a, b) -> Math.abs(a - b) < 0.000001 ? 0 : 1, (a, b) -> !Structs.eq(a, b) ? 1 : 0),
```

Comment:
maybe `!=`? since equal is `==` so it makes sense.

Reply:
the mathematical symbol is = with a slash through it